tox.ini

```diff
@@ -38,6 +38,7 @@ commands =
 
 [testenv:pre_commit]
 deps =
+  {[testenv]deps}
```

Comment:
Can you elaborate why this is necessary?

Reply:
It seemed to me because seed-isort-config is trying to figure out based on `aspy`, but I'm not 100% percent sure, I just observed some inconsistencies, so I thought better put it there to exclude this factor. Could you double check if this is probably a no-op? https://github.com/asottile/aspy.refactor_imports/blob/40b7c18da50859f5fdab11477450693c1614bf54/aspy/refactor_imports/classify.py#L107-L142

Reply:
In my understanding (supported by a green travis), this should not be necessary.

Reply:
Specifically: https://github.com/asottile/aspy.refactor_imports/blob/40b7c18da50859f5fdab11477450693c1614bf54/aspy/refactor_imports/classify.py#L77-L82